`LoggerTools`: Pass `callerMemberName`.

kiminuo opened the pull request:
This makes the log lines contain the place where the logging command was called.

Found when trying to understand why #7080 does not passes tests.

molnard reviewed (changes requested):
NACK: adding this just increases the noise in the logs - file name and line number determine exactly the place of the log event.

kiminuo commented:
It works differently though:

master:
```log
2022-01-31 10:28:40.420 [12] INFO	Arena.Log (352)	Round (e6511f19f2e05b6fddbef4dacda77d56659075009d7cbcc4283c8afc1789b039): 1 alices timed out and removed.
```

This PR:

```log
2022-01-31 10:27:59.334 [12] INFO	Arena.TimeoutAlices (352)	Round (0cedb455a466ada0404fefad63a8f54771f7594b345c07a9113cf9d43f867675): 1 alices timed out and removed.
```

On master, every log line produced by `LoggerTools` report `Arena.Log (line number)`. This PR only makes it more precise by reporting the method (e.g. `Arena.TimeoutAlices (352)`) where the log command was invoked. So this PR does not add anything really, it modifies the log line to, subjectively, better output.

molnard commented:
Yes, you are right I misunderstood the code. ACK
